.changeset/bitter-parrots-end.md

```diff
@@ -0,0 +1,7 @@
+---
+"@skeletonlabs/skeleton-svelte": minor
+"@skeletonlabs/skeleton-react": minor
+---
+
+feat: toast
```

Comment:
Let's be a bit more descriptive here. If we're doing more regular releases the Release tab and changelog needs to help educate users.

> Removed Toast integration in favor of first-party Toast component.